DateMeter.c

```diff
@@ -20,7 +20,11 @@ static void DateMeter_updateValues(Meter* this, char* buffer, int size) {
    time_t t = time(NULL);
    struct tm result;
    struct tm *lt = localtime_r(&t, &result);
-   this->values[0] = lt->tm_year * 365 + lt->tm_yday;
+   this->values[0] = lt->tm_yday;
+   int year = lt->tm_year + 1900;
+   if (((year % 4 == 0) && (year % 100!= 0)) || (year%400 == 0)) {
+      this->total = 366;
```

Comment:
The total is off, when the leap year ended. `this->total` should be properly set in both cases.

Reply:
Fixed